project-base/storefront/connectors/cart/Cart.ts

```diff
@@ -6,102 +6,57 @@ import {
     CartPromoCodeModificationsFragmentApi,
     CartTransportModificationsFragmentApi,
     ListedStoreFragmentApi,
+    Maybe,
     TransportWithAvailablePaymentsAndStoresFragmentApi,
     useCartQueryApi,
 } from 'graphql/generated';
 import { ApplicationErrors } from 'helpers/errors/applicationErrors';
 import { getUserFriendlyErrors } from 'helpers/errors/friendlyErrorMessageParser';
 import { ChangePaymentHandler } from 'hooks/cart/useChangePaymentInCart';
-import { useQueryError } from 'hooks/graphQl/useQueryError';
 import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslationFunction';
 import { useCurrentUserData } from 'hooks/user/useCurrentUserData';
 import { Translate } from 'next-translate';
-import { useMemo } from 'react';
 import { usePersistStore } from 'store/zustand/usePersistStore';
 import { CurrentCartType } from 'types/cart';
 import { GtmMessageOriginType } from 'types/gtm/enums';
-import { CombinedError, OperationContext } from 'urql';
+import { CombinedError } from 'urql';
 
 export const useCurrentCart = (fromCache = true): CurrentCartType => {
     const { isUserLoggedIn } = useCurrentUserData();
     const cartUuid = usePersistStore((store) => store.cartUuid);
     const t = useTypedTranslationFunction();
     const packeteryPickupPoint = usePersistStore((store) => store.packeteryPickupPoint);
 
-    const [result, refetchCart] = useQueryError(
-        useCartQueryApi({
-            variables: { cartUuid },
-            pause: cartUuid === null && !isUserLoggedIn,
-            requestPolicy: fromCache ? 'cache-first' : 'network-only',
-        }),
-    );
-
-    return useMemo(() => {
-        if (result.error !== undefined) {
-            // EXTEND CART ERRORS HERE
-            handleCartError(result.error, t);
-        }
+    const [result, refetchCart] = useCartQueryApi({
```

Comment:
Same as in some other thread I wrote here, a less generic name than 'result' would be nicer. However, if you disagree, you can close this thread.

Reply:
I personally don't care until there are more "results" and since this was just taken from previous implementation I haven't changed this. But it's improved for better readability.